recipe/meta.yaml

```diff
@@ -478,6 +481,43 @@ outputs:
         - test -f $PREFIX/lib/pkgconfig/mkl-sdl.pc  # [linux]
         - if not exist %LIBRARY_LIB%\\pkgconfig\\mkl-sdl.pc exit 1  # [win]
 
+  - name: mkl-static
+    version: {{ mkl_version }}
+    script: repack.sh   # [linux]
+    script: repack.bat  # [win]
+    number: {{ mkl_buildnum|int + dstbuildnum|int }}
+    requirements:
+        {% if win or 'conda-forge' not in my_channel_targets %}
+      run:
+        - {{ pin_subpackage('mkl-include', exact=True) }}
```

Comment:
`mkl-include` and `tbb` may be moved out of if scope

Reply:
agree - but would first integrate this one if ci would pass, and this can be fixed later on